apps/docs/src/guide/wallets/encrypting-and-decrypting-json-wallets.md

```diff
@@ -0,0 +1,27 @@
+# Encrypting and Decrypting JSON Wallet
+
+JSON wallets are a standardized way of storing wallets securely using encryption. They follow a specific schema and are encrypted using a password. This makes it easier to manage multiple wallets and securely store them on disk. This guide will take you through the process of encrypting and decrypting JSON wallets using the Typescript SDK.
```

Comment:
```suggestion
JSON wallets are a standardized way of storing wallets securely. They follow a specific schema and are encrypted using a password. This makes it easier to manage multiple wallets and securely store them on disk. This guide will take you through the process of encrypting and decrypting JSON wallets using the Typescript SDK.
```
- I think that first `using encryption` is redundant as it is superseded by the next sentence
- Should we be referencing the schema it follows?